board/toradex/apalis_imx6/apalis_imx6.c

```diff
@@ -1040,6 +1040,27 @@ static void spl_dram_init(void)
 	udelay(100);
 }
 
+#if defined(CONFIG_SECONDARY_BOOT_RUNTIME_DETECTION) && \
+    defined(CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_USE_SECTOR)
+unsigned long spl_mmc_get_uboot_raw_sector(struct mmc *mmc,
```

Comment:
As another option we can even move that part somwhere to `arch/arm/mach-imx/` to avoid code duplication (as now we have `spl_mmc_get_uboot_raw_sector()` in `boards/freescale/common/mmc.c` and another one here `spl_mmc_get_uboot_raw_sector`)

Reply:
@ricardosalveti @mike-scott

Reply:
Yeah, moving to a more generic place might be better to avoid duplication. We want all of these changes to be generic to the respective imx soc, so makes sense to make it generic code-wise.

Reply:
fixed, please check updates version